### PR TITLE
Cleaned up code for all files and reduced image sizes

### PR DESCRIPTION
**Pull request by thaitwo**
@alexwolfe Can you please take a look at this tiny pull request when you have a chance. Thanks!

### CODE REVIEW

File: public/stylesheets/button.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* -- Default button styles -- */
+.button {
+	font: 14px "Raleway", sans-serif;
```
Comment (alexwolfe) [inline public/stylesheets/button.css]: two spaces for everything sir :)

File: public/stylesheets/button.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	background-color: #64B5F6;
+}
+
+.button-secondary:hover {
+	background-color: #2196F3;
+	transition: all .3s;
+}
+
+.button-primary {
+	color: #fff;
```
Comment (alexwolfe) [inline public/stylesheets/button.css]: 👀

File: public/stylesheets/button.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  border: 1px solid #fff;
+  letter-spacing: 0.2em;
+  text-align: center;
+}
+
+.button-primary:hover {
+	background: #fff;
```
Comment (alexwolfe) [inline public/stylesheets/button.css]: 👀

File: public/stylesheets/footer.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
+footer {
+	padding: 20px 0px 20px 0px;
```
Comment (alexwolfe) [inline public/stylesheets/footer.css]: Indents please :)

File: public/stylesheets/form.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
+.form {
+	width: 400px;
```
Comment (alexwolfe) [inline public/stylesheets/form.css]: Make sure by default you're using "spaces for tabs" check your sublime settings.

File: public/stylesheets/hero.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+  -o-background-size: cover;
+	background-size: cover;
+}
+
+.img-portrait {
+  background: linear-gradient(
+    rgba(0,0,0, 0.7),
+    rgba(0,0,0, 0.7)),
+    url("/images/portrait/portrait-5.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed.

File: public/stylesheets/hero.css [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.img-gallery {
+	background: linear-gradient(
+    rgba(0,0,0, 0.7),
+    rgba(0,0,0, 0.7)),
+		url("/images/landscape/landscape-11.jpg") no-repeat center center fixed;
+	-webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed.

File: public/stylesheets/hero.css [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.img-la-night {
+	background: linear-gradient(
+    rgba(0,0,0, 0.7),
+    rgba(0,0,0, 0.7)),
+		url("/images/la-night.jpg") no-repeat center center fixed;
+	-webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed.

File: public/stylesheets/hero.css [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+  background-size: cover;
+}
+
+.img-la-night {
+	background: linear-gradient(
+    rgba(0,0,0, 0.7),
+    rgba(0,0,0, 0.7)),
+		url("/images/la-night.jpg") no-repeat center center fixed;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: One line please, background: .... all on one line.

File: public/stylesheets/hero.css [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.img-landscape {
+  background: linear-gradient(
+    rgba(0,0,0,0.7),
+    rgba(0,0,0,0.7)),
+    url("/images/landscape/landscape-1.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed.

File: public/stylesheets/hero.css [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+    rgba(0,0,0,0.4)),
+    url("/images/half-dome-night.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.img-landscape {
+  background: linear-gradient(
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: one line please

File: public/stylesheets/hero.css [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+  font: 64px "Raleway", sans-serif;
+  font-weight: 800;
+  color: #fff;
+  text-align: center;
+  text-shadow: 0 0 8px rgba(0, 0, 0, 0.4);
+  padding: 0 0 30px 0;
+}
+
+.img-half-dome {
+  background: linear-gradient(
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: one line please

File: public/stylesheets/hero.css [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+    rgba(0,0,0,0.4),
+    rgba(0,0,0,0.4)),
+    url("/images/half-dome-night.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed.

File: public/stylesheets/hero.css [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+    rgba(0,0,0, 0.7)),
+    url("/images/portrait/portrait-5.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.img-food {
+  background: linear-gradient(
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: one line please 👯

File: public/stylesheets/hero.css [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+    rgba(0,0,0, 0.7),
+    rgba(0,0,0, 0.7)),
+    url("/images/food/food-5.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed. 🌵

File: public/stylesheets/hero.css [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 50%;
+  background: #fff;
+  -webkit-transform: skew(0deg, 3deg);
+  -moz-transform: skew(0deg, 3deg);
+  -ms-transform: skew(0deg, 3deg);
+  -o-transform: skew(0deg, 3deg);
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed. 🍔

File: public/stylesheets/hero.css [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+  content: '';
+  position: absolute;
+  right: 0;
+  height: 100%;
+  width: 50%;
+  background: #fff;
+  -webkit-transform: skew(0deg, -3deg);
+  -moz-transform: skew(0deg, -3deg);
+  -ms-transform: skew(0deg, -3deg);
+  -o-transform: skew(0deg, -3deg);
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: you can remove all the -prefix classes, no longer needed. 🍨

File: routes/index.js [hunk 18]
```diff
@@ -16,4 +16,24 @@ router.get('/contact', function(req, res, next) {
 	res.render('contact', { title: 'CONTACT'});
 });
 
+/* GET gallery page */
+router.get('/gallery', function(req, res, next) {
+	res.render('gallery', { title: 'GALLERIES'});
```
Comment (alexwolfe) [inline routes/index.js]: Technically you can make the the titles Capitalized not All Caps (ex. Galleries) and css can all cap the content for you.

File: views/contact.ejs [hunk 19]
```diff
@@ -1,2 +1,15 @@
...
+	<br>Or simply just to say hi.</p>
+	<div class="clear"></div>
+</article>
+
+<article class="grid">
+	<div class="col-6">
+	<% include ../views/includes/_form %>
+	</div>
+	<img class="col-6" src="/images/marlette.jpg">
+	<div class="clear"></div>
```
Comment (alexwolfe) [inline views/contact.ejs]: I believe this is not necessary since grid has a clear attached to it using :after.

File: views/includes/_form.ejs [hunk 20]
```diff
@@ -0,0 +1,14 @@
+<form class="form">
+		<ul>
+			<li>
+				<input type="text" name="name" placeholder="YOUR NAME">
```
Comment (alexwolfe) [inline views/includes/_form.ejs]: woaaah, those tabs are killing me... ✈️

File: views/includes/_gallery-nav.ejs [hunk 21]
```diff
@@ -1,7 +1,50 @@
...
+		<img src="/images/portrait.jpg">
+		<figcaption>
+			<h2>
+				PORTRAIT
+			</h2>
+			<p>VIEW GALLERY</p>
+			<a href="/gallery/portrait">View More</a>
+		</figcaption>
+	</figure>
+	</figure>
```
Comment (alexwolfe) [inline views/includes/_gallery-nav.ejs]: extra closing `</figure>`

File: views/includes/_gallery-nav.ejs [hunk 22]
```diff
@@ -1,7 +1,50 @@
...
+		<img src="/images/landscape.jpg">
+		<figcaption>
+			<h2>
+				LANDSCAPE
+			</h2>
+			<p>VIEW GALLERY</p>
+			<a href="/gallery/landscape">View More</a>
+		</figcaption>
+	</figure>
+	<figure class="image-card">
```
Comment (alexwolfe) [inline views/includes/_gallery-nav.ejs]: Add a space between different items (ie cards)

File: views/includes/_gallery-nav.ejs [hunk 23]
```diff
@@ -1,7 +1,50 @@
...
+
+		<a class="image-card bg-portrait" href="#">
+			<strong>PORTRAIT</strong>
+		</a>
+
+		<a class="image-card bg-food" href="#">
+			<strong>FOOD</strong>
+		</a>
+		<div class="clear"></div>
+</nav>
```
Comment (alexwolfe) [inline views/includes/_gallery-nav.ejs]: Remove this commented code.

File: views/includes/_top-nav.ejs [hunk 24]
```diff
@@ -1,9 +1,9 @@
 <nav class="top-nav">
-	<h1><a href="/">THAI TU PHOTOGRAPHY</a></h1>
+	<h1><a href="/">thaitwo<em>photography</em></a></h1>
 
 	<ul>
 		<li><a href="/contact">CONTACT</a></li>
 		<li><a href="/about">ABOUT</a></li>
-		<li><a href="/gallery">GALLERY</a></li>
+		<li><a href="/gallery">GALLERIES</a></li>
```
Comment (alexwolfe) [inline views/includes/_top-nav.ejs]: Again, all caps in the html not necessary, css can do that yo 🎾

**Comment by alexwolfe** [comment]
Ok, looking really good. Just had a few comments. You'll need to update/fix the spacing on your files (convert tabs to spaces and make sure the indents are consistent). Other than that, looks good. Make those updates an ping me... 👍

**Comment by thaitwo** [comment]
Thanks for the edits! Everything should be updated now. :ramen:

File: public/stylesheets/hero.css [hunk 25]
```diff
@@ -0,0 +1,112 @@
+.hero {
+	display: block;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: whoops spacing! 👯

File: public/stylesheets/hero.css [hunk 26]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.img-landscape {
+  background: linear-gradient(rgba(0,0,0,0.7), rgba(0,0,0,0.7)), url("/images/landscape/landscape-1.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+
+.img-about {
+	background: linear-gradient(rgba(0,0,0, 0.7), rgba(0,0,0, 0.7)), url("/images/landscape/landscape-8.jpg") no-repeat center center fixed;
+	background-size: cover;
```
Comment (alexwolfe) [inline public/stylesheets/hero.css]: More space 🚶

File: routes/index.js [hunk 27]
```diff
@@ -3,17 +3,37 @@ var router = express.Router();
...
 
 /* GET contact page */
 router.get('/contact', function(req, res, next) {
-	res.render('contact', { title: 'CONTACT'});
+	res.render('contact', { title: 'Contact'});
+});
+
+/* GET gallery page */
+router.get('/gallery', function(req, res, next) {
+	res.render('gallery', { title: 'Galleries'});
```
Comment (alexwolfe) [inline routes/index.js]: spacing sir 🎩

File: views/gallery-portrait.ejs [hunk 28]
```diff
@@ -0,0 +1,21 @@
+<%- include('../views/includes/_hero', {background:"img-portrait"}); %>
+
+<div class="grid">
+	<a class="col-4" href="/images/portrait/portrait-13.jpg"><img src="/images/thumbnails-portrait/portrait-thumbnail-13.jpg"></a>
```
Comment (alexwolfe) [inline views/gallery-portrait.ejs]: spacing again sir 🔮

**Pull request merged**